site/pages/our-work/case-study/camden-market/style.css

```diff
@@ -84,19 +84,20 @@
 }
 
 .header__container {
-  max-width: 760px;
-  margin: 0 0 auto auto;
+  max-width: 1160px;
+  margin: auto;
   position: relative;
-  width: 100%;
+  width: calc(100% - 120px);
```

Comment:
Nice spot, this explains why there wasn't a correct margin for certain screen sizes